lib/cli/src/frameworks/lit/js/Button.js

```diff
@@ -0,0 +1,77 @@
+import { LitElement, html, css } from 'lit';
+import { styleMap } from 'lit/directives/style-map.js';
+
+export class StorybookButton extends LitElement {
+  static get styles() {
+    return css`
+      .storybook-button {
+        font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+        font-weight: 700;
+        border: 0;
+        border-radius: 3em;
+        cursor: pointer;
+        display: inline-block;
+        line-height: 1;
+      }
+      .storybook-button--primary {
+        color: white;
+        background-color: #1ea7fd;
+      }
+      .storybook-button--secondary {
+        color: #333;
+        background-color: transparent;
+        box-shadow: rgba(0, 0, 0, 0.15) 0 0 0 1px inset;
+      }
+      .storybook-button--small {
+        font-size: 12px;
+        padding: 10px 16px;
+      }
+      .storybook-button--medium {
+        font-size: 14px;
+        padding: 11px 20px;
+      }
+      .storybook-button--large {
+        font-size: 16px;
+        padding: 12px 24px;
+      }
+    `;
+  }
+
+  // TODO: currently decorators are not reflected https://github.com/Polymer/lit-html/issues/1476
```

Comment:
Given that this file will be shipped to users, do we want to leave this TODO for them?

Reply:
Curious too.  This is a nice to know as this format is a little bit weird to see when using decorators else where.  Maybe we can document this somewhere and remove the TODOs? or keep this available so users can track the issue.

Reply:
Bad copy paste 🙈